portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalSmokeUpgrade.testcase

```diff
@@ -5,7 +5,7 @@ definition {
 	property database.types = "db2,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.main.component.name = "Database Upgrade Tool";
```

Comment:
My understanding of having the testray component name as "Database Upgrade Tool" would be that we are using the tool to perform the Upgrade, and then it would be related to the component, since we have other files performing the Upgrade with the Auto Upgrade and therefore referring to the "Upgrade Framework".

In the case of changing this way, shouldn't we also change the` @component-name`?

cc/ @vicnate5

Reply:
We are using the tool to perform the upgrade. But the point of these tests to check the upgrade data and overall success. So we care more about the upgrade as a whole, not just the tool that executes the upgrade. 

I want to focus "Database Upgrade Tool" component on tests for the tool itself (the cli, script, property generation) and then I kept the Upgrade Report tests under the tool since that gets generated after the tool is run.

Reply:
You are right I am missing the `component-name` change. Will update that.

Reply:
Nice, thanks for the explanation, also Upgrade Report is being implemented also for Auto Upgrade at [LPS-158740](https://issues.liferay.com/browse/LPS-158740).